src/main/java/com/clickhouse/kafka/connect/sink/data/convert/SchemalessRecordConvertor.java

```diff
@@ -28,7 +28,7 @@ public Record convert(SinkRecord sinkRecord) {
         int index = 0;
         map.forEach((key,val) -> {
                     fields.add(new Field(key.toString(), index, Schema.STRING_SCHEMA));
-                    data.put(key.toString(), new Data(Schema.Type.STRING, val == null ? null : val.toString()));
+                    data.put(key.toString(), new Data(Schema.STRING_SCHEMA, val == null ? null : val.toString()));
```

Comment:
Since we also touched the schema-less convertor, let's add some tests for it also to cover complex types of nesting

Reply:
Sure, I've replicated the same complex types to present in Schemaless tests.
Btw, weird they weren't covered before, because it was supposed to work.

Anyway, it's also covered now!